src/edgepi/eeprom/eeprom_constants.py

```diff
@@ -13,6 +13,23 @@
 SCg0AAIA/FQAAAAAaCg0AAIA/FQAAAAAiCg0AAIA/FQAAAAAqCg0AAIA/FQAAAAAyCg0AAIA/FQAAAAA6Cg0AAIA/FQAAAABCCg\
 0AAIA/FQAAAAA='
 
+"""
+Defaul EEPROM Dataclass
+dac_calib_params.dac_ch_1~8.gain = 1 offset = 0
+adc1_calib_params.adc_ch_1~8, diff1~4.gain = 1 offset = 0
+adc2_calib_params.adc_ch_1~8, diff1~4.gain = 1 offset = 0
+rtd_calib_params.rtd.gain = 1 offset = 0
+rtd_calib_params.rtd_resistor = 2000
+tc_calib_params.tc_B,E,J,K,N,R,S,T.gain = 1 offset = 0
```

Comment:
This is hard for me to read. I don't really understand what is going on here. Maybe it needs to be less condensed and we can use indentation to mark children of the same thing. Not sure if that makes sense.

Reply:
[a4aea98](https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/369/commits/a4aea98c56ec92b8ec443c416225e247803b332f)